### ISSUE TITLE
postUpload.sh permission issue

### ISSUE THREAD

**Issue by only1chunts** [gigascience/gigadb-website#1292]
**Describe the bug**
 I've just run the postUpload script on a newly created dataset on the beta server. It appears to run and provides on-screen output, but it makes no changes to the database and it has not saved any files where it says it has saved them? the full command line output is pasted below:
Is it possible its a permissions issue?

> 
> Full paste of output:
> $ sudo /home/centos/postUpload.sh 102402
> 
> * About to update files' size for 102402
> nb. changes: 1
> 
> Done with updating files' size for 102402. Nb of successful changes saved in file: /home/centos/uploadLogs/updating-file-size-102402.txt
> 
> * About to check that file urls are valid for 102402
> | URL | Issue |
> | https://ftp.cngb.org/pub/gigadb/pub/10.5524/102001_103000/102402/readme_102402.txt | Resource cannot be downloaded, not found or forbidden (4xx) |
> 
> Done with checking that file urls are valid for 102402. Invalid Urls (if any) are save in file: /home/centos/uploadLogs/invalid-urls-102402.txt
> 
> * About to update files' MD5 Checksum as file attribute for 102402
> No file found at https://ftp.cngb.org/pub/gigadb/pub/10.5524/104001_105000/102402/102402.md5
> No file found at https://ftp.cngb.org/pub/gigadb/pub/10.5524/103001_104000/102402/102402.md5
> No file found at https://ftp.cngb.org/pub/gigadb/pub/10.5524/102001_103000/102402/102402.md5
> No file found at https://ftp.cngb.org/pub/gigadb/pub/10.5524/101001_102000/102402/102402.md5
> No file found at https://ftp.cngb.org/pub/gigadb/pub/10.5524/100001_101000/102402/102402.md5
> No 102402.md5 file could be found for dataset DOI 102402
> Done with updating files' MD5 Checksum as file attribute for 102402. Process status is saved in file: /home/centos/uploadLogs/updating-md5checksum-102402.txt
> 
> * About to create the README file for 102402
> [DOI] 10.5524/102402
> 
> [Title] Supporting data for "MuLan-Methyl - Multiple Transformer-based Language
> Models for Accurate DNA Methylation Prediction"
> 
> [Release Date] 2023-05-31
> 
> [Citation] Zeng, W; Gautam, A; Huson, DH (2023): Supporting data for
> "MuLan-Methyl - Multiple Transformer-based Language Models for Accurate DNA
> Methylation Prediction" GigaScience Database. https://dx.doi.org/10.5524/102402
> 
> [Data Type] Software,Workflow
> 
> [Data Summary] Transformer-based language models are successfully used to
> address massive text-related tasks. DNA methylation is an important epigenetic
> mechanism and its analysis provides valuable insights into gene regulation and
> biomarker identification. Several deep learning-based methods have been proposed
> to identify DNA methylation and each seeks to strike a balance between
> computational effort and accuracy. Here, we introduce MuLan-Methyl, a
> deep-learning framework for predicting DNA methylation sites, which is based on
> five popular transformer-based language models. The framework identifies
> methylation sites for three different types of DNA methylation, namely
> N6-adenine, N4-cytosine, and 5-hydroxymethylcytosine. Each of the employed
> language models is adapted to the task using the “pre-train and fine-tune”
> paradigm. Pre-training is performed on a custom corpus of DNA fragments and
> taxonomy lineages using self-supervised learning. Fine-tuning aims at predicting
> the DNA-methylation status of each type. The five models are used to
> collectively predict the DNA methylation status. We report excellent performance
> of MuLan-Methyl on a benchmark dataset. Moreover, we show that the model
> captures characteristic differences between different species that are relevant
> for methylation. This demonstrates that language models can be successfully
> adapted to applications in biological sequence analysis and that joint
> utilization of different language models improves model performance.
> Mulan-Methyl is open source and we provide a web server that implements the
> approach.
> 
> [File Location] https://ftp.cngb.org/pub/gigadb/pub/10.5524/102001_103000/102402/
> 
> [File name] - [File Description]
> mulan-methyl-main.zip  -  Archive copy of the GitHub repository https://github.com/husonlab/mulan-methyl, downloaded 26 May 2023. MuLan-Methyl - Multiple Transformer-based Language Models for Accurate DNA Methylation Prediction. Released under Apache 2.0 license. Please visit the GitHub repository for the most recent updates.
> readme_102402.txt  -
> 
> [License]
> All files and data are distributed under the CC0 1.0 Universal (CC0 1.0) Public
> Domain Dedication (https://creativecommons.org/publicdomain/zero/1.0/), unless
> specifically stated otherwise, see http://gigadb.org/site/term for more details.
> 
> [Comments]
> 
> [End]
> 
> Done with creating the README file for 102402. The README file is saved in file: /home/centos/uploadLogs/readme-102402.txt
> [chrish@ip-10-99-0-32 ~]$ cd uploadDir/
> [chrish@ip-10-99-0-32 uploadDir]$ ls -l
> total 60
> -rw-rw-r--. 1 centos centos    16 May 17 06:38 invalid-urls-100006.txt
> -rw-r--r--. 1 centos root       0 May 26 09:29 javac.log
> -rw-r--r--. 1 centos root   43957 May 26 09:29 java.log
> -rw-rw-r--. 1 centos centos  2528 May 17 06:39 readme-100006.txt
> -rw-rw-r--. 1 centos centos    15 May 17 06:38 updating-file-size-100006.txt
> -rw-rw-r--. 1 centos centos   739 May 17 06:39 updating-md5checksum-100006.txt
> 
> [chrish@ip-10-99-0-32 uploadDir]$ less /home/centos/uploadLogs/updating-file-size-102402.txt
> /home/centos/uploadLogs/updating-file-size-102402.txt: Permission denied

**Additional context**
@rija says 

> A permission issue on the postUpload.sh script has been identified.
> 
> The generated files should have gone back to your personal uploadDir
> directory like the java log file generated by the datasetUpload.sh script.
